samples/cpp/chat_sample/chat_sample.cpp

```diff
@@ -4,15 +4,14 @@
 #include "openvino/genai/llm_pipeline.hpp"
 
 int main(int argc, char* argv[]) try {
-    if (2 != argc) {
-        throw std::runtime_error(std::string{"Usage: "} + argv[0] + " <MODEL_DIR>");
+    if (2 != argc && 3 != argc) {
+        throw std::runtime_error(std::string{"Usage: "} + argv[0] + " <MODEL_DIR> <DEVICE>(optional)");
```

Comment:
Don't change that. The idea is to minimize the number of cmd args. You can extend the comment instead with NPU

Reply:
Returned back anyway